src/Compilers/CSharp/Portable/Parser/SyntaxParser.cs

```diff
@@ -494,7 +494,7 @@ protected SyntaxToken EatTokenAsKind(SyntaxKind expected)
                 return ct;
             }
 
-            var replacement = CreateMissingToken(expected, this.CurrentToken.Kind, reportError: true);
+            var replacement = CreateMissingToken(expected, this.CurrentToken.Kind, reportError);
             return AddTrailingSkippedSyntax(replacement, this.EatToken());
```

Comment:
>AddTrailingSkippedSyntax [](start = 19, length = 24)

It feels strange to have no error in a syntax tree that has skipped tokens. If we want to consume keywords as calling conventions, perhaps we should relax/remove kind restriction on it in the syntax.xml and consume the token normally. Or "reinterpret" the keyword token into an identifier token with the text and position of the keyword without creating any skipped tokens.